### DISCUSSION TITLE
How do you

### DISCUSSION THREAD

**Discussion by audrey-mcknight** [melink14/rikaikun#1033] [Q&A]
![image](https://user-images.githubusercontent.com/83185853/170276453-f1f632d8-0187-49de-9244-8b87ccedeff9.png)
So as pictured above, the text is being extracted from an image. So rikaikun chunks it all together and doesn't let you look at each character or word individually, but still shows the definitions of the first few characters in the chunk. But it doesn't have the whole thing, and cuts it off because it ran out of room. 
How do I see the full translation? Maybe add a scrollbar

**Comment by melink14** [comment]
Thanks for the question. There's not been too much development for this feature since it's not used much but I'll try to explain a bit about it.

First, I want to be clear that the text that is being translated isn't extracted from the image directly but instead is read from any `alt` or `title` atttributes attached to the image. That is to say, the creator of the webpage has added some explanatory text that rikaikun can read to show the translations as best as it can.

I clarify that because sometimes the website creators only provide a snippet of text alongside their images which means there's nothing rikaikun can do to display more text since it doesn't do any kind of image OCR scanning. Do you have a link to where this image appeared? I can double check if there's more data available.

On the other hand, there is a built in limit as well and I think the suggestion to add scrolling (via j/k keys like in other cases) isn't a bad idea. Generally, the feature has been most useful for one or two word images but there's no reason it can't work in longer sentence cases as well.
